core/src/mindustry/Vars.java

```diff
@@ -141,6 +141,10 @@
     public static final int multicastPort = 20151;
     /** multicast group for discovery.*/
     public static final String multicastGroup = "227.2.7.7";
+    /** extra time after the end of the game. */
+    public static final int roundExtraTime = 12;
```

Comment:
What is the point of placing either of these variables here? They're still static final, and they have nothing to do with "shared" variables. Either make them `Config` numbers, or don't move them.